chosww/idrc
# feat: multilingual CMS and front-end, simplify project children

Resolves #628.


## Description

- Implements multilingual CMS configuration for projects (fix #628)
- Switches to Eleventy Navigation for project child page handling
- Initial front-end implementation of multilingual projects (#629)

## Steps to test

1. Add a French version of one of the projects.
2. Run `npm start`.
3. Visit the project page and switch between English and French.
4. Log in to the CMS locally and observe French/English editing.

## Additional information

Not applicable.

## Related issues

#626, #627
